test/extensions/filters/http/wasm/test_data/metadata_cpp.cc

```diff
@@ -22,19 +22,19 @@ static RegisterContextFactory register_ExampleContext(CONTEXT_FACTORY(ExampleCon
 
 void ExampleRootContext::onTick() {
   google::protobuf::Value value;
-  if (MetadataResult::Ok != nodeMetadataValue("wasm_node_get_key", &value)) {
-    logDebug("onTick metadata error");
+  if (auto r = nodeMetadataValue("wasm_node_get_key", &value); r != WasmResult::Ok) {
```

Comment:
Nit: This isn't Go, could you split this into 2 lines?

Reply:
Done.

You don't like the new c++17 syntax?  It doesn't leave the WasmResult hanging in the surrounding scope.

Reply:
Envoy is still using C++14.